Mirror/Runtime/NetworkTransform.cs

```diff
@@ -886,21 +886,15 @@ void FixedUpdateServer()
 
         bool CheckVelocityChanged()
         {
-            if (transformSyncMode == TransformSyncMode.SyncRigidbody2D)
+            switch (transformSyncMode)
             {
-                if (m_RigidBody2D && m_VelocityThreshold > 0)
-                {
+                case TransformSyncMode.SyncRigidbody2D when m_RigidBody2D && m_VelocityThreshold > 0:
```

Comment:
case when is great,  but it is only available in C# 7,   which we have decided not to use until 2018 goes LTS.

Reply:
Wait, sorry, i've copied wrong code